packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/AnnotationService.java

```diff
@@ -0,0 +1,38 @@
+package software.uncharted.terarium.hmiserver.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import software.uncharted.terarium.hmiserver.entities.Annotation;
+import software.uncharted.terarium.hmiserver.repository.AnnotationRepository;
+
+import java.util.List;
+
+@Service
+@Slf4j
+public class AnnotationService {
```

Comment:
Since there's nothing fancy going on in here you can use the repository directly you in the controller and delete this

Reply:
If its all the same, I think i may keep `AnnotationService` just to have it mirror events' structure